jujugui/static/gui/src/app/jujulib/charmstore.js

```diff
@@ -184,6 +184,9 @@ var module = module;
           location: extraInfo['bzr-url']
         }
       };
+      if (bundleMeta && bundleMeta.Description) {
+        processed.description = bundleMeta.Description;
```

Comment:
Or just
```javascript
if(bundleMeta) {
  processed.description = bundleMeta.Description || '';
}
``` 
?
So that we always have a description as a string?